fdbclient/StorageServerInterface.h

```diff
@@ -159,7 +159,7 @@ struct WatchValueRequest {
 struct GetKeyValuesReply : public LoadBalancedReply {
 	constexpr static FileIdentifier file_identifier = 1783066;
 	Arena arena;
-	VectorRef<KeyValueRef> data;
+	VectorRef<KeyValueRef, true> data;
```

Comment:
Nit:  With an enum class or named constexpr var instead of bool literal here I think this would look nicer: `Vector<KeyValueRef, kStringSerialized>` e.g.